market_maker/ws/ws_thread.py

```diff
@@ -291,16 +296,17 @@ def __on_message(self, message):
         except:
             logger.error(traceback.format_exc())
 
-    def __on_open(self):
+    def __on_open(self, arg):
         logger.debug("Websocket Opened.")
+        logger.info(arg)
 
     def __on_close(self):
         logger.info('Websocket Closed')
         self.exit()
 
-    def __on_error(self, error):
+    def __on_error(self, error, arg):
         if not self.exited:
-            self.error(error)
+            self.error(arg)
```

Comment:
I'm not sure I understand this change, although I have seen it mentioned in a couple of old issues:
https://github.com/BitMEX/api-connectors/issues/311

It could be to do with how you're running it locally or versions of libs you have - in any case can we please split this out into its own PR since it doesn't look specifically related to the changes you're making here

Reply:
In fact - have you tried [this](https://github.com/BitMEX/sample-market-maker/pull/235/files)?

Reply:
Thanks. Addressed in 38e8ac6e0ad72214af9609b6a254f23286c3a194

Reply:
> In fact - have you tried [this](https://github.com/BitMEX/sample-market-maker/pull/235/files)?

Yes, didn't help for me.